src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
Sounds like a plan. What I would like to retain is the ability to combine telemetry protocols that only use Tx on the FC with serialRx protocols that only use Rx on the same port.

My question for @martinbudden was about that, since `TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK` was introduced to list these combinable telemetry protocols, and I am not sure if CRSF should be on this list.